services/getstream/updateUser.js

```diff
@@ -2,7 +2,7 @@ const stream = require('getstream');
 const client = stream.connect(process.env.API_KEY, process.env.SECRET);
 
 module.exports = async (userId, data) => {
-  client
+  await client
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The use of `await` with a promise that uses `.then()` is redundant. You should either use async/await or promises, but not both together. Here's how you can refactor this to use async/await only:

```diff
-  await client
+  try {
+    const result = await client.user(userId).update(data);
+    // handle result here
+  } catch (error) {
+    // handle error here
+  }
```
This way, you're properly handling any potential errors that might occur during the update operation.


<!-- This is an auto-generated comment by OSS CodeRabbit -->

Reply:
please remove await